util/regexp-assemble/data/942240.data

```diff
@@ -3,8 +3,30 @@
 
 ##!+ i
 
-alter\s*?\w+.*?character\s+set\s+\w+
-alter\s*?\w+.*?char\s+set\s+\w+
-[\"'`];*?\s*?waitfor\s+time\s+[\"'`]
-[\"'`];*?\s*?waitfor\s+delay\s+[\"'`]
-[\"'`];.*?:\s*?goto
+##! Helpers
+##!> template alter alter\s*?\w+.*?
+
+##!> template set \s+set\s+\w+
+
+##!> template ticks [\"'`]
+
+##!> template waitfor *?\s*?waitfor\s+
```

Comment:
Looking at this expression, I have the feeling that the `.` got lost at some point. I don't see the reason to match `;*`, which would be included in `.*` anyway. By making this change, you can also move the `;` up to after `{{ticks}}`, below.

```suggestion
##!> template waitfor .*?\s*?waitfor\s+
```

Reply:
No, the problem was that I split the following regex into two parts: `;*?` a lazy match of `;`.
I tried to fix it again and hopefully the tests will succeed this time.